src/components/Header/index.jsx

```diff
@@ -153,7 +153,7 @@ const GordonHeader = ({ onDrawerToggle }) => {
 
   return (
     <section className={styles.gordon_header}>
-      <AppBar className={styles.app_bar} position="static">
+      <AppBar className={styles.app_bar} position="static" color="primary">
```

Comment:
According to the [`AppBar` API docs](https://mui.com/material-ui/api/app-bar/), `"primary"` is the default value of the `color` prop, so I think this isn't needed.
```suggestion
      <AppBar className={styles.app_bar} position="static">
```